<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parameters</class>
 <widget class="QWidget" name="parameters">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Set parameters</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../media/interfaz/opciones.png</normaloff>../media/interfaz/opciones.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(89, 126, 143);
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Initial position of the system, recommended value: middle of map. Default: (400,400)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Initial Position (integer):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Size of map in cm. Default: 800x800.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Size of map (integer):</string>
   </property>
  </widget>
  <widget class="QLabel" name="x_lev">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Variation of X between to tests to optimize position in SLAM. Recommended values: 1 to 3.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Variation of X (integer):</string>
   </property>
  </widget>
  <widget class="QLabel" name="a_lev">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Variation of Angle(theta) between to test to optimize position in SLAM. Recommended values: 0.4 to 3.0.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Variation of angle (float):</string>
   </property>
  </widget>
  <widget class="QLabel" name="y_lev">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Variation of Y between to tests to optimize position in SLAM. Recommended values: 1 to 3. &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Variation of Y (integer):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Weight of the occupation assigned to the last value of a pixel. A high value means that the new information has low importance. Recommended values: 0.4 to 0.7 Default: 0.7&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Weight of last pixel value (float):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_x">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_wi">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_x_var">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_y_var">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_a_var">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_we">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_accept">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>480</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(98, 195, 177);</string>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="b_cancel">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>480</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(224, 147, 147);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_y">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_h">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Baudrate (integer):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sensor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Port which the system is connected to. Default: /dev/ttyUSB0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Port (string):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_baud">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_port">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Map and SLAM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>All fields in this window are optional</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_levmarq">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Levenberg Marquardt algorithm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Algorithm applied to solve SLAM&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Algorithm</string>
   </property>
  </widget>
  <widget class="QComboBox" name="algorithm">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Brute force</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Levenberg Marquardt</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="range_x">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;From -(value/2) to (value/2)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="range_angle">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>330</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QLabel" name="y_brute">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;From -(value/2) to (value/2). Measure: cm. Default: 20&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Range to estimate Y (integer):</string>
   </property>
  </widget>
  <widget class="QLabel" name="angle_brute">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>300</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;From -(value/2) to (value/2). Measure: degrees. Default: 60&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Range to estimate angle (integer):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="range_Y">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>300</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;From -(value/2) to (value/2)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QLabel" name="x_brute">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;From -(value/2) to (value/2). Measure: cm. Default: 20&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Range to estimate X (integer):</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_brute">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Brute force algorithm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Whether to ignore the human while mapping or not (ignoring the human means there is an angle range that will not be mapped)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Ignore human: </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ignore_human">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>text_x</tabstop>
  <tabstop>text_y</tabstop>
  <tabstop>text_wi</tabstop>
  <tabstop>text_h</tabstop>
  <tabstop>text_we</tabstop>
  <tabstop>algorithm</tabstop>
  <tabstop>text_x_var</tabstop>
  <tabstop>text_y_var</tabstop>
  <tabstop>text_a_var</tabstop>
  <tabstop>range_x</tabstop>
  <tabstop>range_Y</tabstop>
  <tabstop>range_angle</tabstop>
  <tabstop>text_baud</tabstop>
  <tabstop>text_port</tabstop>
  <tabstop>b_cancel</tabstop>
  <tabstop>b_accept</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
